### PR TITLE
Pass cluster directly in Round Robin load balancing policy

### PR DESCRIPTION
**Pull request by Kahbazi**

### CODE REVIEW

File: src/ReverseProxy/LoadBalancing/RoundRobinLoadBalancingPolicy.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ internal sealed class RoundRobinLoadBalancingPolicy : ILoadBalancingPolicy
                 return null;
             }
 
-            var counter = _counters.GetOrCreateValue(context.GetClusterState());
```
Comment (Tratcher) [inline src/ReverseProxy/LoadBalancing/RoundRobinLoadBalancingPolicy.cs:25]: It looks like this was the last call to GetClusterState. Can you remove it?

Comment (Kahbazi) [reply src/ReverseProxy/LoadBalancing/RoundRobinLoadBalancingPolicy.cs:25]: Done.

**Review by Tratcher — APPROVED** [review]

**Comment by Tratcher** [comment]
Thanks

**Comment by Kahbazi** [comment]
@Tratcher Does this PR need the breaking-change label?

**Comment by Tratcher** [comment]
I guess that was a public API we removed 😁. Applied.

**Comment by franklixuefei** [comment]
We were calling `context.GetClusterState()`. What should we call now?

**Pull request merged**